views/tour.dt

```diff
@@ -34,10 +34,10 @@ block content
 				- if (sourceCodeEnabled)
 					button.btn.btn-primary(ng-click="run()")
 						i.fa.fa-play(aria-hidden="true")
-						span Run
+						span #{translations["run"]}
 					button.btn.btn-default(ng-click="reset()")
 						i.fa.fa-undo(aria-hidden="true")
-						span Reset
+						span #{translations["reset"]}
```

Comment:
the keys should be a bit more verbose for better maintainability in the future as `"reset"` can mean many things in different contexts. For example `"editor.reset"`